hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -531,10 +531,11 @@
 
             - While Ibex is executing, trigger a debug request by programming `dmcontrol.haltreq`
               via JTAG.
-            - Ibex steps into interactive debug to store a value in SRAM.
-            - Program the program buffer through JTAG with code to store a value in SRAM and return
-              back to normal program execution using dret.
-            - After Ibex jumps back to the program execution, check the stored value in the SRAM.
+            - Ibex steps into interactive debug to execute instructions in SRAM.
+            - Load instruction data into the SRAM via JTAG.  X-ref'ed with chip_sw_sram_execution.
```

Comment:
i think specifically for this test, it's not a huge deal how we get the instruction data into SRAM. It's fine if it comes from the software also.  The more critical part is jumping the ibex to said code via JTAG.

If it's easier to get the instruction data in via JTAG, great! If not, i think it's fine to re-use the infrastructure in the existing test also

Reply:
Good point. loading instruction via SW is much easier and we just copy the same approach from chip_sw_sram_execution test.